aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -170,6 +171,17 @@
    */
   public static final String S3FILEIO_SESSION_TOKEN = "s3.session-token";
 
+  /**
+   * Enable to make S3FileIO, to make cross-region call to the region specified in the ARN of an access point.
+   * <p>
+   * When set, the client can make a cross-region call to the region specified in the ARN of an access point,
+   * otherwise an exception will be thrown if cross-region call is tried.
```

Comment:
I think it would be more clear to say "By default, attempting to use an access point in a different region will throw an exception. When enabled, this property is allows using access points in other regions".